tests/settings.yaml

```diff
@@ -1,24 +1,42 @@
 locations:
   input-dir: in/
   output-dir: out/
-  genome-dir: genome/
+  genome-fasta: genome/sample.fasta
 
 general:
   assembly: hg19
+  use_bwameth: True
+  # bismark is gold standard and uses sensible defaults
+  use_bismark: True
   methylation-calling:
     minimum-coverage: 1
     minimum-quality: 10
+    keep-Dups: Auto                 
   differential-methylation:
     cores: 1
     qvalue: 0.05
-    treatment-groups:
-      - ['0', '1']
-      - ['0', '2']
     annotation:
-      cpgIsland_bedfile: pigx_work/refGenome/cpgIslandExt.hg19.bed.gz 
-      refGenes_bedfile:  pigx_work/refGenome/refGene.hg19.bed.gz
+      cpgIsland-bedfile: pigx_work/refGenome/cpgIslandExt.hg19.bed.gz 
+      refGenes-bedfile: pigx_work/refGenome/refGene.hg19.bed.gz
       webfetch:   no
 
+DManalyses:
+  # The names of analyses can be anything but they have to be unique
+  # for each combination of case control group comparisons.
+
+  med1vsh2o:
+    # If multiple sample names are provided, they must be separated by comma.
+    treatment_sample_groups: "MED1"
+    control_sample_groups: "H2O"
+
+  multipleTreat:
+    treatment_sample_groups: "MED2,MED1"
+    control_sample_groups: "H2O"
+    
+  withinGroup:
+    treatment_sample_groups: "MED2,MED1,H2O"
+    control_sample_groups: "MED2,MED1,H2O"
```

Comment:
same as above on use of arrays.

Reply:
I would leave that open until we have #160 .